invenio_app_rdm/theme/assets/js/react_invenio_deposit/DepositController.js

```diff
@@ -29,11 +29,11 @@ export class DepositController {
       if (payload.errors) {
         console.log("create errors", payload.errors);
       } else {
-        const newURL = payload.data.links.edit
+        const newURL = payload.data.links.edit;
         window.history.replaceState(undefined, "", newURL);
       }
     }
-    return this.apiClient.save(payload);
+    return this.apiClient.save(payload.data);
```

Comment:
Thanks for catching that one. I had fixed the same issue with `publish` yesterday, but didn't check save.